File: python/PRAK403-2310817310004-MSAMILRENDYNORSALEH.py
angka1, angka2 = map(int, input("").split())

terbesar = angka1 if angka1 > angka2 else angka2
terkecil = angka1 if angka1 < angka2 else angka2

if angka1 > angka2:
    for i in range(terbesar, terkecil - 1, -1):
        print(f"{i} {angka1 + angka2 - i}", end="")
        if i > terkecil:
            print(" - ", end="")
else:
    for i in range(terkecil, terbesar + 1):
        print(f"{i} {angka1 + angka2 - i}", end="")
        if i < terbesar:
            print(" - ", end="")

print()

File: c/PRAK403-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    printf("");
    scanf("%d %d", &angka1, &angka2);

    int terbesar = (angka1 > angka2) ? angka1 : angka2;
    int terkecil = (angka1 < angka2) ? angka1 : angka2;

    if (angka1 > angka2) {
        for (int i = terbesar; i >= terkecil; i--) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i > terkecil) {
                printf(" - ");
            }
        } 
    } else {
    for (int i = terkecil; i <= terbesar; i++) {
        printf("%d %d", i, angka1 + angka2 - i);
        if (i < terbesar) {
            printf(" - ");
            }
        }
    }
    printf("\n");

    return 0;
}
